lib/orderbook/OrderBook.ts

```diff
@@ -826,6 +826,22 @@ class OrderBook extends EventEmitter {
     return order;
   }
 
+  public removeOwnOrders = () => {
+    const removedOrderLocalIds = [];
+    const onHoldOrderLocalIds = [];
+
+    for (const localId of this.localIdMap.keys()) {
+      try {
+        this.removeOwnOrderByLocalId(localId, true);
+        removedOrderLocalIds.push(localId);
+      } catch (ex) {
+        onHoldOrderLocalIds.push(localId);
```

Comment:
`removeOwnOrderByLocalId` doesn't throw an error when the order has a hold, instead it will return a non-zero number (the method comments explain in more detail). So rather than try/catching here, you should just check whether it returns 0 or not and add it to one of the two lists accordingly.

Reply:
True that, sorry for overlooking it!